diff --git a/node_modules/jest-environment-jsdom/build/index.js b/node_modules/jest-environment-jsdom/build/index.js
index 77176ba..1644673 100644
--- a/node_modules/jest-environment-jsdom/build/index.js
+++ b/node_modules/jest-environment-jsdom/build/index.js
@@ -102,6 +102,7 @@ class JSDOMEnvironment {
     (0, _jestUtil().installCommonGlobals)(global, config.globals); // TODO: remove this ASAP, but it currently causes tests to run really slow
 
     global.Buffer = Buffer; // Report uncaught errors.
+    global.Uint8Array = Uint8Array;
 
     this.errorEventListener = event => {
       if (userErrorListenerCount === 0 && event.error) {
