diff --git a/node_modules/jest-webextension-mock/dist/setup.js b/node_modules/jest-webextension-mock/dist/setup.js
index cd0c2eb..3132e44 100644
--- a/node_modules/jest-webextension-mock/dist/setup.js
+++ b/node_modules/jest-webextension-mock/dist/setup.js
@@ -72,7 +72,8 @@ var runtime = {
   getURL: jest.fn(function (path) {
     return path;
   }),
-  openOptionsPage: jest.fn()
+  openOptionsPage: jest.fn(),
+  getManifest: jest.fn(() => ({manifest_version: 3}))
 };
 
 // https://developer.chrome.com/extensions/tabs
